audit-service/README.md

```diff
@@ -58,7 +75,7 @@ The application exposes one REST endpoint for the purpose of debugging:
     - accepts either an `AuditSuccessMessage` or an `AuditErrorMessage` as the request body
```

Comment:
The core of the work flow for this is the AkkaRunnableGraph and all of it could be best described by walking though the runner method.  I think that there needs to be a section that describes this.

Reply:
This part is not about the RunnableGraph as this is how a developer can debug the service. I have added something about the RunnableGraph within the Kafka Interface section above here.